tests/syft/core/node/domain/domain_test.py

```diff
@@ -106,3 +112,22 @@ def test_domain_request_accepted() -> None:
     )
 
     assert RequestStatus.Accepted == response
+
+
+def test_domain_is_for_me_exception() -> None:
+    domain_1 = Domain(name="remote domain")
+
+    with pytest.raises(Exception):
+        msg = SyftMessage()
+        domain_1.message_is_for_me(msg)
+
+    # @syft_decorator(typechecking=True)
```

Comment:
Q: Could we add a TODO + issue for this?

Reply:
This has been removed.